src/test/java/com/segment/analytics/android/integrations/appboy/AppboyTest.java

```diff
@@ -177,7 +177,6 @@ public void testIdentifyFields() {
     verify(mAppboyUser).setCustomUserAttribute("string", "value");
     //verifyNoMoreAppboyUserInteractions();
     verify(mAppboy, Mockito.times(1)).changeUser("userId");
-    verify(mAppboy, Mockito.times(15)).getCurrentUser();
```

Comment:
Was this failing because of this PR or another reason?

Reply:
Nop, it was already failing in master. Most probably something to do with a change in the implementation when we added the user trait's cache... it's not something that should be verified anyway, IMO 😶